doc/src/build/move/build-test.md

```diff
@@ -247,19 +250,9 @@ The code of the new functions is self-explanatory and uses struct
 creation and Sui-internal modules (`TxContext` and `Transfer`) in a
 way similar to what we have seen in the previous sections. The
 important part is for the entry functions to have correct signatures
-as described [earlier](index.md#entry-functions). In order for this code to

```

Comment:
I couldn't figure out why this exposition wasn't necessary anymore -- the entry functions do introduce a reference to `TxContext`.  Having said that, one clean-up you can now make is to remove the unreferenced `_ctx: &mut TxContext` parameter from `sword_transfer` as entry functions are now allowed to omit that parameter if it goes unused.

Reply:
Because I was not required to do this part.
Earlier in the tutorial at [defining the package](https://docs.sui.io/build/move/write-package#defining-the-package) I had already imported TxContext. Mentioning that you first need to import a struct before using it was already clear at this stage in the tutorials.
I'll leave the `_ctx: &mut TxContext` parameter in for now.